Algebras/DotBracket/Parts/algpart_dotBracket_unique.gap

```diff
@@ -0,0 +1,147 @@
+  string sadd(Subsequence lb,string e) {
```

Comment:
hm, why did you add `_unique` to the filename? The unique property only stems from the choice function `h` but has nothing to do with all the implemented algebra functions here. We might need a phone call to figure it out efficiently, but it looks to me like this file here should be identical to "algpart_dotBracket_basic.gap", i.e. is not really needed.